packages/travesty/travesty.0.1.3/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+synopsis: "Monadically traversable containers"
+description:
+  "'Travesty' is a library for defining containers with monadic traversals,
+   inspired by Haskell's Traversable typeclass.  It sits on top of Jane Street's
+   Core library ecosystem."
+maintainer: "Matt Windsor <m.windsor@imperial.ac.uk>"
+authors: "Matt Windsor <m.windsor@imperial.ac.uk>"
+license: "MIT"
+doc: "https://MattWindsor91.github.io/travesty/"
+homepage: "https://MattWindsor91.github.io/travesty"
+bug-reports: "https://github.com/MattWindsor91/travesty/issues"
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune" {build}
+  "ppx_deriving" {build}
+  "ppx_expect" {build}
+  "ppx_jane" {build}
+  "ppx_sexp_message" {build}
```

Comment:
We decided to stop marking the ppx rewriters as `{build}` packages as they may carry directly or indirectly associated runtime libs. I am going to remove the flag and merge. Please report it upstream